### PR TITLE
Like/dislike

### PR DESCRIPTION
**Pull request by NamrataRaikwar2002**
-like/unlike functionality

-user can like a vieo
-unlike a video

### CODE REVIEW

**Review by MonikaJhingan — APPROVED** [review]
The code looks good to be merged.

File: src/App.css [hunk 1]
```diff
@@ -34,19 +38,37 @@ input {
...
+}
+
+.emptyHeading {
+  color: var(--secondary-color);
+}
+
+.modalInapp{
+  display: flex;
+  justify-content: center;
+  /* align-items: center; */
```
Comment (tewarimohit) [inline src/App.css:65]: Remove comments..

File: src/Component/Playlist/Playlist.css [hunk 2]
```diff
@@ -0,0 +1,62 @@
+.playlistDiv {
+    position: fixed;
+    /* margin-top: 10rem; */
```
Comment (tewarimohit) [inline src/Component/Playlist/Playlist.css:3]: remove comments..

**Review by tewarimohit — APPROVED** [review]
Good to merge.... Code looks clean.

**Review by pradipgharge — APPROVED** [review]
Code looks clean and good to merge !

**Pull request merged**